cat/c11.cat

```diff
@@ -20,7 +25,7 @@ let cacq = ACQ | (SC & (R | F)) | ACQ_REL | (F & CON)
 let crel = REL | (SC & (W | F)) | ACQ_REL
 let ccon = R & CON
 
-// To be compatible with reads from uninit memory, we use the default definition of fr
+// To be compatible with reads from uninit memory, we use our default definition of fr
 //let fr = rf^-1 ; mo
 
```

Comment:
I think we can get rid of this altogether now that we have a copy of the original cat model. Also, I would use `po` and `co` ... in the original one I was using `sb` and `mo` to keep it as faithful to the one I took from herd's repo as possible

Reply:
Do you really want to change everything? We can do it, but it makes it harder to compare both models.
That's also why I kept the original code in comments to more easily see where something was changed.

Reply:
I kind of disliked to have all that commented code, but I agree that by keeping them we can simply use diff and easy check the changes between those models, so let's do it that way